rotateList.py

```diff
@@ -0,0 +1,38 @@
+class Solution(object):
+    def rotateRight(self, head, k):
+        """
+        :type head: ListNode
+        :type k: int
+        :rtype: ListNode
+        """
+        if not head or k == 0:
+            return head
+        count = 0
+        curNode = head
+        while curNode:
+            count += 1
+            curNode = curNode.next
+        
+        target = count - k % count
+        if target == 0:
```

Comment:
`k % count` would return a number between `0` and `count - 1`.
`target` would never be equal to 0.
How could this work?